pkgs/top-level/all-packages.nix

```diff
@@ -14063,6 +14063,10 @@ with pkgs;
 
   trunk-io = callPackage ../development/tools/trunk-io { };
 
+  trunk-ng = callPackage ../development/tools/trunk-ng {
+    inherit (darwin.apple_sdk.frameworks) CoreServices Security;
+  };
```

Comment:
Try moving it to the new `by-name` hierarchy, see https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name